### ISSUE TITLE
Code bug in gaussian-splatting/utils/render_utils.py

### ISSUE THREAD

**Issue by White-YC** [umangi-jain/gaussiancut#4]
The code `mask[mask > 0] = 1.0` in the multiview case would select both foreground and background, and lead to wrong segmentation results.
```
        elif mask_type == 'multiview':
            mask_base_path = os.path.join(scene_path, "multiview_masks")
            masks_all = sorted(os.listdir(mask_base_path))

            all_viewpoints = scene.getTrainCameras().copy() + \
                scene.getTestCameras().copy()

            images_names = []
            for index in range(len(masks_all)):
                image_name = masks_all[index].split('.')[0]
                images_names.append(image_name)
            select_viewpoints = [[] for _ in range(len(images_names))]

            for index in range(len(all_viewpoints)):
                for index_image, image_name in enumerate(images_names):
                    if all_viewpoints[index].image_name == image_name:
                        select_viewpoints[index_image] = all_viewpoints[index]
                        break

            assert len(masks_all) == len(
                select_viewpoints
            ), "Number of masks and viewpoints do not match for all. Please check mask names!"
            for index, _ in enumerate(
                    tqdm(select_viewpoints,
                         desc="Coarse rasterization progress")):
                # skip rasterization for test cameras
                if select_viewpoints[index] in scene.getTestCameras():
                    continue
                mask_path = os.path.join(mask_base_path, masks_all[index])
                mask_image = Image.open(mask_path)
                transform = transforms.ToTensor()
                mask_tensor = transform(mask_image)

                _, height, width = select_viewpoints[index].original_image.shape
                mask_tensor = F.interpolate(
                    mask_tensor.unsqueeze(0).float(), size=(height, width),
                    mode="bilinear", align_corners=False).squeeze(0)
                mask = mask_tensor.repeat(3, 1, 1).contiguous().cuda()
                mask[mask > 0] = 1.0

                gaussians_for_source.apply_weights(select_viewpoints[index],
                                                   weights, weights_cnt, mask)
                gaussians_for_sink.apply_weights(select_viewpoints[index],
                                                 weights_sink, weights_cnt_sink,
                                                 1 - mask)
```
